src/main/scala/com/databricks/spark/avro/DefaultSource.scala

```diff
@@ -210,6 +221,8 @@ private[avro] class DefaultSource extends FileFormat with DataSourceRegister {
 private[avro] object DefaultSource {
   val IgnoreFilesWithoutExtensionProperty = "avro.mapred.ignore.inputs.without.extension"
 
+  val AVRO_SCHEMA = "avroSchema"
```

Comment:
Maybe use `AvroSchema` to keep consistent with the other constant defined above.